app/kuma-dp/pkg/dataplane/metrics/metrics_format_mapper.go

```diff
@@ -0,0 +1,127 @@
+package metrics
+
+import (
+	"math"
+
+	io_prometheus_client "github.com/prometheus/client_model/go"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/sdk/metric/metricdata"
+)
+
+func FromPrometheusMetrics(appMetrics []*io_prometheus_client.MetricFamily) []metricdata.Metrics {
```

Comment:
Nit: I am not sure how many different formats we will have, but we could have an interface `Mapper` with method `Map` and each type could implement it. Maybe it's not important now because it looks like we only support Prometheus now.

Reply:
I thought about this, but now we have only mapping from prometheus, and I don't know if we will add anything in the future, and if we need any other format we can add this interface easily later on